packages/better-php-doc-parser/src/PhpDocInfo/PhpDocInfoFactory.php

```diff
@@ -90,7 +90,8 @@ public function createFromNode(Node $node): ?PhpDocInfo
             $tokens = [];
             $phpDocNode = new AttributeAwarePhpDocNode([]);
         } else {
-            $content = $node->getDocComment()->getText();
+            $content = $node->getDocComment()
+                ->getText();
```

Comment:
@TomasVotruba , these kind of fixes by `fix-cs` look weird to me... I don't like :)
I could agree that it makes sense when the line is too long, or to make line break after the second chained method. :)

Reply:
I see. The rule of 1 method chain call per line is more consistent and logical to read.

Same way 1 item per array item line.

```php
[
    'one' => 'two',
    'one' => 'two',
    'one' => 'two',
]
```